datafusion/sql/src/unparser/expr.rs

```diff
@@ -683,6 +686,16 @@ mod tests {
                 ScalarUDF::new_from_impl(DummyUDF::new()).call(vec![col("a"), col("b")]),
                 r#"dummy_udf("a", "b")"#,
             ),
+            (
+                Expr::Like(Like {
```

Comment:
I was going to suggest using https://docs.rs/datafusion/latest/datafusion/logical_expr/enum.Expr.html#method.like but I think that doesn't allow setting esape_char. This looks good to me.